core/src/main/scala/org/apache/spark/deploy/worker/Worker.scala

```diff
@@ -821,7 +820,7 @@ private[deploy] class Worker(
   }
 
   private def generateWorkerId(): String = {
-    workerIdPattern.format(createDateFormat.format(new Date), host, port)
+    workerIdPattern.format(Worker.DATE_TIME_FORMATTER.format(new Date().toInstant), host, port)
```

Comment:
Is it possible to directly use `Instant.now()` instead of `new Date().toInstant`?